openstack_plugin/resources/compute/server.py

```diff
@@ -679,6 +709,43 @@ def _get_network_name(nic_object, client_config):
     return net_name
 
 
+def _get_security_groups_ids(security_groups, client_config):
+    """
+    This method will return all security groups ids so they can be used
+    later on for attaching to servers and the reason for getting the ids
+    instead of names because there is an issue with openstacksdk
+    :param security_groups: List of security groups
+            [
+                {
+                'name': 'security-group-name-1'
+                },
+                {
+                'name': 'security-group-name-1'
+                }
+            ]
+    :param dict client_config: Openstack configuration required to connect
+    to API
+    :return: List of security groups
+    [
+        {
+        'name': '2830b6f1-801c-4973-84e2-7772f7e20e2a'
+        },
+        {
+        'name': '8347a687352e37767f93305cca090357'
+        }
+    ]
+    """
+    sg_list = []
+    security_group = OpenstackSecurityGroup(client_config=client_config,
+                                            logger=ctx.logger)
+    for sg in security_groups:
+        remote_sg = security_group.find_security_group(sg.get('name'))
+        sg_list.append({
+            'name': remote_sg.id
```

Comment:
Do we want to call it 'name' or 'id'?

Reply:
Good question, we can change it to `id` to make it more consistent with the value begin sent to the API